src/components/ChatRoom/index.js

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useRef, useState } from "react";
+import { useCollectionData } from "react-firebase-hooks/firestore";
+import { AuthorAvatar, FormSubmit, FormInput, Message, MessageForm, MessagesList, MessageText, MessageTime, Wrapper } from "./styled";
+import { nanoid } from "nanoid";
+
+const ChatRoom = ({ firebase, firestore, auth }) => {
+    const messagesRef = firestore.collection('messages');
+    const getMessagesByTime = messagesRef.orderBy('createdAt').limit(25);
+    const [messages] = useCollectionData(getMessagesByTime);
+
+    const [input,setInput] = useState('');
+    const onInputChange = (e) => {
+        setInput(e.target.value);
+    }
+    useEffect(() => {
+    },[messages])
```

Comment:
Not sure if that useEffect is needed at all

Reply:
Napisałeś na czacie, że masz rozwiązanie do scrolla za pomocą tego useEffect, więc pewnie przytadne.